### DISCUSSION TITLE
Pending PRs

### DISCUSSION THREAD

**Discussion by vladmandic** [vladmandic/sdnext#258] [Announcements]
I've created an initial backlog of interesting PRs that are still pending in A1111:
<https://github.com/vladmandic/automatic/pulls>

If you have any other PRs that you'd like to see merged, comment on this thread

**Comment by kohweiteck** [comment]
Thank you Vladmandic for your awesome work! Wishing nothing but best of health for you and loved ones!

**Comment by Nacurutu** [comment]
Are u planning to merge these PRs or is the list just for people to merge them if they want?

Also, what about the collab with @anapnoe and his UI? its going to happen?

**Reply by vladmandic** [reply]
i plan to merge them as soon as possible, i just wanted to go over the open prs on a1111 and create a priority list.
the only issue is i'm a bit overloaded at the moment.
same applies to collab with @anapnoe, its on me to get things moving.

**Reply by Nacurutu** [reply]
take ur time... no pressure...
